sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1157,8 +1343,9 @@ void update(node& node);
 ----
 
 | Updates an executable graph node that corresponds to `node`. `node` must be a
-kernel execution node. Kernel arguments and the ND-range of the node will be
-updated inside the executable graph to reflect the current values in `node`.
+kernel execution node. Kernel arguments and the command-group function of the 
+node will be updated inside the executable graph to reflect the current values
+in `node`.
```

Comment:
If we are being specific here (and below) should we still not mention that the range will be updated? Since that can be modified independently of the CGF.

Reply:
I'm not sure. I think that by adding `dynamic_command_group`, there is potentially confusing terminology in the spec.

The way I see it, a kernel node is essentially a command-group function. So updating only the range using `update_nd_range()` would be included implicitly in the wording.  I find it confusing to specify command-group functions and nd_range as 2 separate things.

But this can also be confusing because we haven't been using the term "command-group function" to define nodes previously. The definition of node is:

```quote
A node in a queue recorded graph represents a command-group submission to the
device associated with the queue being recorded. Nodes are constructed from
the command-group functions (CGF)...
```

I guess I could be specific by saying:

"The command-group function of the node will be updated inside the executable graph to reflect the current values in `node`. This includes the kernel arguments, the kernel binary, the kernel nd-range, accessors and dynamic_parameters".

or just ommit command-group function all together and say

```quote
The kernel arguments, the kernel binary, the kernel nd-range, 
accessors and dynamic_parameters will be updated inside the executable graph 
to reflect the current values in `node`.".
```

Do you think this would be any better?

Reply:
We haven't used the term "kernel binary" so far I think, so I'm a bit wary that we haven't defined it, and I'm also not sure if "accessors" are different from kernel parameters here.

However, I do think the layout "The command-group function of the node will be updated inside the executable graph to reflect  the current values in `node`. This includes ... " is a good way to phrase things.

Reply:
> and I'm also not sure if "accessors" are different from kernel parameters here.

I agree I don't think accessors need to be mentioned specifically.

> However, I do think the layout "The command-group function of the node will be updated inside the executable graph to reflect the current values in `node`. This includes ... " is a good way to phrase things.

Yeah I think including ND-range in the list of things which will be updated here is fine.

Reply:
I have implemented this suggestion and changed mentions of kernel binary to kernel function which is a term that seems to be well defined in the sycl spec.